examples/post_processing.py

```diff
@@ -0,0 +1,85 @@
+"""
+Example full-screen post processing.
+"""
+
+import time
+
+import numpy as np
+import imageio
+import pyshader
+import pygfx as gfx
+
+from PyQt5 import QtWidgets
+from wgpu.gui.qt import WgpuCanvas
+
+
+app = QtWidgets.QApplication([])
+
+
+class NoisyPostProcessingStep(gfx.renderers.wgpu.PostProcessingStep):
+    def __init__(self, device):
+        super().__init__(
+            device,
+            fragment_shader=noise_fragment_shader,
+            uniform_type=noise_uniform_type,
+        )
+
+    def render(self, src, dst):
+        self._uniform_data["time"] = time.perf_counter()
+        self._uniform_data["noise"] = 0.03
+        super().render(src, dst)
+
+
+noise_uniform_type = pyshader.Struct(time=pyshader.f32, noise=pyshader.f32)
+
+
+@pyshader.python2shader
+def noise_fragment_shader(
+    v_texcoord: (pyshader.RES_INPUT, 0, "vec2"),
+    s_sam: (pyshader.RES_SAMPLER, (0, 0), ""),
+    t_tex: (pyshader.RES_TEXTURE, (0, 1), "2d f32"),
+    u_render: (pyshader.RES_UNIFORM, (0, 2), noise_uniform_type),
+    out_color: (pyshader.RES_OUTPUT, 0, "vec4"),
+):
+    val = t_tex.sample(s_sam, v_texcoord)
+
+    # todo: use spirv builtin random number gen
+    xy = v_texcoord.xy
+    noise = fract(sin(xy @ vec2(12.9898, 78.233) + u_render.time) * 43758.5453)
+    noise = u_render.noise * noise
+    val += vec4(noise, noise, noise, 1.0)
+    out_color = val  # noqa - shader output
+
+
+canvas = WgpuCanvas()
+renderer = gfx.renderers.WgpuRenderer(canvas)
+scene = gfx.Scene()
+
+renderer.post_processing_steps.append(NoisyPostProcessingStep(renderer.device))
```

Comment:
How about `renderer.postfx` for short?

Reply:
mmm, I can imagine situations where you want to add a post-filter for a while and then remove it later. We could add API for that, but a list object already has this API :)

That said, I have not given this API much thought. At first the SSAA step was also part of this list, but things worked better to make it "builtin". Maybe doing post-processing effects should be done completely different still. Let's make an issue to track this.

Reply:
#72

Reply:
Fair enough! In that case I still suggest `renderer.postfx` as a shorter name for the list property :)

Reply:
Haha, agreed!